spec/std/socket/socket_spec.cr

```diff
@@ -87,6 +87,8 @@ describe Socket, tags: "network" do
 
     expect_raises(IO::TimeoutError) { server.accept }
     expect_raises(IO::TimeoutError) { server.accept? }
+
+    server.close
```

Comment:
**suggestion:** Put this in an `ensure` block so it gets closed even when the spec fails.
```suggestion
  ensure
    server.try &.close
```